Vagrantfile

```
@@ -47,6 +50,26 @@ Vagrant.configure(2) do |config|
     do
         echo unset $LC  >> /home/vagrant/.bashrc
     done
+
+    # Install VirtualBox guest additions
+    cd /tmp
+    VBOX_VERSION=$1
+    VBOX_ADDITIONS_URL="http://download.virtualbox.org/virtualbox/VER/VBoxGuestAdditions_VER.iso"
+    sudo wget -q -c ${VBOX_ADDITIONS_URL//VER/$VBOX_VERSION} -O VBoxGuestAdditions_$VBOX_VERSION.iso
```

What you want is `-nv` then.  With `-q` you hide the errors so you don't know why it failed.